core/src/main/java/org/p2p/core/token/TokenExtensions.kt

```diff
@@ -27,3 +27,7 @@ data class TokenExtensions(
 fun List<Token.Active>.filterTokensForWalletScreen(): List<Token.Active> {
     return filter { it.tokenExtensions.isTokenVisibleOnWalletScreen == true }
 }
+
+fun List<Token.Active>.filterIsUSDCToken(): List<Token.Active> {
+    return filter { it.tokenSymbol.equals("USDC", ignoreCase = true) }
```

Comment:
is the `Token.isUSDC` not working because the symbol can be in lower case? At least, you can use `Constants.USDC_SYMBOL`